default/stringtables/de.txt

```diff
@@ -8148,53 +8177,68 @@ BAD_DEFENSE_TROOPS_DESC
 −	Schlechte Boden[[metertype METER_TROOPS]]: 50%
 
 AVERAGE_DEFENSE_TROOPS_DESC
-'''	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%'''
+⋄	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%
 
 GOOD_DEFENSE_TROOPS_DESC
-+	Gute Boden[[metertype METER_TROOPS]]: 150%.
++	Gute Boden[[metertype METER_TROOPS]]: 150%
 
 GREAT_DEFENSE_TROOPS_DESC
-++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%.
+++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%
 
 ULTIMATE_DEFENSE_TROOPS_DESC
-+++	Perfekte Boden[[metertype METER_TROOPS]]: 300%.
++++	Perfekte Boden[[metertype METER_TROOPS]]: 300%
 
 BAD_DETECTION_DESC
-−	Schlechte [[metertype METER_DETECTION]]: −20 Malus.
+−	Schlechte [[metertype METER_DETECTION]]: −20 Malus
 
 GOOD_DETECTION_DESC
-+	Gute [[metertype METER_DETECTION]]: +25 Bonus.
++	Gute [[metertype METER_DETECTION]]: +25 Bonus
 
 GREAT_DETECTION_DESC
-++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus.
+++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus
 
 ULTIMATE_DETECTION_DESC
-+++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus.
++++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus
 
 BAD_STEALTH_DESC
-−	Schlechte [[metertype METER_STEALTH]]: −20 Malus.
+−	Schlechte [[metertype METER_STEALTH]]: −20 Malus
 
 AVERAGE_STEALTH_DESC
-'''	Mittelmäßige [[metertype METER_STEALTH]].'''
+⋄	Mittelmäßige [[metertype METER_STEALTH]]
 
 GOOD_STEALTH_DESC
-+	Gute [[metertype METER_STEALTH]]: +20 Bonus.
++	Gute [[metertype METER_STEALTH]]: +20 Bonus
 
 GREAT_STEALTH_DESC
-++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus.
+++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus
 
 ULTIMATE_STEALTH_DESC
-+++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus.
++++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus
 
 BAD_POPULATION_DESC
 −	Niedrige Bevölkerung 75%
 
 AVERAGE_POPULATION_DESC
-'''	Normale Bevölkerung'''
+⋄	Normale Bevölkerung
 
 GOOD_POPULATION_DESC
 +	Hohe Bevölkerung 125%
 
+GREAT_ASTEROID_INDUSTRY_DESC
++	Guter Asteroidenbergbau: + 0.2 [[metertype METER_INDUSTRY]] je [[metertype METER_POPULATION]] bei Industriefokus in Systemen mit besetzten Asteroidengürteln
+
+LIGHT_SENSITIVE_DESC
+−	Lichtempfindlich: [[metertype METER_POPULATION]] stark reduziert in Sternensystemen der Farbe [[STAR_BLUE]], weniger stark bei Sternen der Farbe [[STAR_WHITE]]
+
+TELEPATHIC_DETECTION_DESC
++	Telepathisches Gespür: kann nahe bevölkerte Welten fühlen
+
+PRECOGNITIVE_DETECTION_DESC
++	Vorwissendes Gespür: kann Planeten jenseits der Sternenstraßen erfühlen
+
+COMMUNAL_VISION_DESC
+⋄	Gemeinsame Sicht: geteilte Sichtbarkeit in der Spezies unabh. d. Reichszugehörigkeit
```

Comment:
"Gemeinsame Wahrnehmung: Sichtbarkeit wird innerhalb derselben Spezies geteilt, unabhängig von Reichszugehörigkeit." The last part after the comma isn't actually there in the English string (maybe should?), but in German it's probably a good idea to add this.

That aside, "Sichtbarkeit wird ... geteilt" isn't really proper German, but I too can't think of a better way to say this.

Reply:
soo .. maybe let the not-quite-perfect-but-understandable german stand instead of no translation of that segment? you see in professional work life i do computer related translation stuff professionally .. there i hate to fall short of perfection as well. but in this case i'd say, if the 100% is not achievable, then the 75% is better than zero .. maybe?

Reply:
> in the English string (maybe should?)
yes indeed, i got that over in another pr with the english minor tweakage stuff, it says their now smth like 'irregardless of affiliation'

Reply:
> but in this case i'd say, if the 100% is not achievable, then the 75% is better than zero

Yes, definitely. My statement was only meant as a comment, not an objection. As long as we can't come up with something better, we'll work with what we _can_ come up with...